src/components/reps/error-handler.jsx

```diff
@@ -101,10 +99,7 @@ export function trimStack(e) {
   const outputFrames = []
 
   for (const frame of frames) {
-    if ((frame.functionName !== undefined &&
-         frame.functionName.startsWith(runCodeWithLanguage.name)) ||
-        (frame.functionName === undefined &&
-         frame.fileName !== 'cell')) {
+    if (frame.fileName !== undefined || frame.fileName !== 'cell') {
```

Comment:
@mdboom We lose the line number with this, but it's possible that we didn't have it in the first place because `cell` isn't a thing (possibly?). Wondering what we might need to do to get some kind of line number if we want it when code is eval'd.

Reply:
Yeah, line numbers used to work.  I think this is still broken.

Reply:
I was testing on master before -- this seems to behave differently on the jsmd branch...

Reply:
Yeah, we haven't addressed this issue yet. Not sure how to atm.